src/qt/listctrl.cpp

```diff
@@ -911,10 +949,20 @@ class wxQtVirtualListModel : public wxQtListModel
 
 class wxQtListTreeWidget : public wxQtEventSignalHandler< QTreeView, wxListCtrl >
 {
+    using BaseClass = wxQtEventSignalHandler< QTreeView, wxListCtrl >;
+
+    // Data type passed to EmitListEvent() as extra data which is essentially
+    // a pair of two ints with these meanings:
+    // first  = column or first (de)selected row
+    // second = column width or last (de)selected row
```

Comment:
Would it be much more difficult to declare this as a struct with the named fields describing their meanings? I find the code using `pair` much more difficult to use than using named fields.